composer.json

```diff
@@ -11,12 +11,14 @@
         }
     ],
     "require": {
+        "ext-sqlite3": "*",
+        "ext-pdo_sqlite": "*",
```

Comment:
ext-sqlite3 is not a dependency for this package

Reply:
Is it just pdo_sqlite then?

Reply:
`pdo_sqlite` as well. This package doesn't use sqlite

Reply:
Yes it does - that's why I added it. The unit tests use a sqlite database connection.

Reply:
Maybe it should go in require-dev?